src/interfaces/object-validator.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectValidationResult } from "../types/object-validation-result";
+import { ObjectValidator } from "utilities/validation/object-validator";

```

Comment:
Looks like this import is using the `baseUrl` from `tsconfig.json` which will break in the packaged JS. Please update this to be relatively pathed like the import above it.

Also, nitpick, but we usually add new lines between imports and any class/interface definitions

Reply:
Removed { ObjectValidator } import entirely since it was unused and replaced it with an empty line for formatting before the interface definition.